src/tests/ftest/daos_test/rebuild.yaml

```diff
@@ -66,6 +72,10 @@ daos_tests:
     test_rebuild_28: DAOS_Rebuild_28
     test_rebuild_29: DAOS_Rebuild_29
     test_rebuild_30: DAOS_Rebuild_30
+    test_rebuild_31: DAOS_Rebuild_31
```

Comment:
Unfortunately test_rebuild_31 currently hangs. I discussed with @liuxuezhao and that failure is related to some overall rebuild error case handling that needs refinement, https://daosio.atlassian.net/browse/DAOS-15847.

He gave me an idea to try with test_rebuild_31 using a different type of fault injection. I can check that out separately. In the meantime, maybe we could (if the tests are able to pass) enable tests 32-35.

Reply:
I've added skip list entries for test_rebuild_31 and test_rebuild_32.